pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -229,10 +229,6 @@ func (b *OFBridge) DeleteGroup(id GroupIDType) error {
 	if ofctrlGroup == nil {
 		return nil
 	}
-	g := &ofGroup{bridge: b, ofctrl: ofctrlGroup}
```

Comment:
Why remove line 232 to 235, but not directly call OFBridge.DeleteGroup(gID) in the caller?

This change might cause group still exists in OVS when calling OFBridge.DeleteGroup but not sending another separate OF message to remove group.